pandas/core/aggregation.py

```diff
@@ -0,0 +1,177 @@
+from collections import defaultdict
+from functools import partial
+from typing import Any, DefaultDict, Sequence
+
+from pandas.core.dtypes.common import is_dict_like, is_list_like
+
+import pandas.core.common as com
+from pandas.core.indexes.api import Index
+
```

Comment:
can you add a doc-string to the module, describing the purpose of it 

e.g. functions to handle multiple named and lambda kwarg aggregations in groupby and dataframe/series.agg

Reply:
ahh, nice idea, added!